themes/codefor-theme/layouts/partials/head.html

```diff
@@ -30,4 +30,5 @@
   {{ printf `<link rel="%s" type="%s" href="%s" title="%s" />` .Rel .MediaType.Type .Permalink $.Site.Title | safeHTML }}
   {{ end -}}
 
+  {{ partial "analytics.html" . }}
```

Comment:
IMO this would be better suited just before the closing `</body>` tag to not block loading unnecessarily